test/test.ts

```diff
@@ -283,10 +283,10 @@ describe('getTextFromHtml', () => {
 });
 
 describe('formatTimestamp', () => {
-  it('should return the formatted date and time in IST', () => {
+  it('should return the formatted date and time according to timezone', () => {
```

Comment:
let's avoid using the word timezone or region anywhere for this function docs..this function is only converting unix timestamp to another format... @divyjainjd

Reply:
@jainarpit014 updated the above for this functions docs